RELEASE/scripts/autoscend/auto_powerlevel.ash

```diff
@@ -398,6 +398,28 @@ boolean LX_freeCombats(boolean powerlevel)
 		adv_done = autoAdv(1, $location[An Unusually Quiet Barroom Brawl]);
 		if(adv_done) return true;
 	}
+	
+	if(auto_haveBurningLeaves() && get_property("auto_remainingFlamingLeafFights").to_int() > 0) {
+		auto_log_debug("LX_freeCombats is summoning leaf monsters");
+		int remainingLeafFights = get_property("auto_remainingFlamingLeafFights").to_int();
+
+		while(remainingLeafFights > 0 && item_amount($item[inflammable leaf]) >= 11){
+			remainingLeafFights--;
```

Comment:
We want a conditional here to put on the tearaway pants which gives adventures from leaves.

```
if (auto_haveTearawayPants()) {
    addBonusToMaximize($item[Tearaway Pants],400);
}
```

We may have to later add a manual call to pre_adv to make sure that gets appropriately equipped.